src/Plot.tsx

```diff
@@ -101,6 +103,8 @@ export default function Plot(props: PlotProps) {
             width={width}
             height={height}
             style={{ ...defaultSvgStyle, ...svgStyle }}
+            id={id}
+            className={className}
```

Comment:
should be `svgId` and `svgClassName` for consistency with the already existing `svgStyle`.